config.ini

```diff
@@ -57,6 +57,8 @@
       nameserver = ""
   [[storage]]
     size = "120"
+    # set below attribute to 'false' if virtual disk needs to be created and attached to PIM partition
+    use_physical_volume = true
```

Comment:
let's not add any more config here, add this also in virtual-disk section and use only size param here.